packages/sitecore-jss-nextjs/src/tests/next-setup.js

```diff
@@ -0,0 +1,6 @@
+﻿process.env = {
```

Comment:
Is this extra setup file required if we are doing setup in beforeEach?

Reply:
So before setting up this file the `process.env__NEXT_IMAGE_OPTS` was coming to be undefined in the test file.
I figured out that we need to set up this file to be able to access the next config variables in our test files.

Reply:
Got it